<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorSettingsDialog</class>
 <widget class="QDialog" name="EditorSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>648</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>648</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="ui_autoCompletionGroup">
       <property name="title">
        <string>Auto Completion</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="ui_autoCompletionEnabledCheckBox">
            <property name="text">
             <string>Enabled</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>66</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Source:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QCheckBox" name="ui_autoCompletionAPISourceCheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>API</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ui_autoCompletionDocumentSourceCheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Document</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Threshold:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="ui_autoCompletionThresholdSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="ui_autoIndentGroup">
       <property name="title">
        <string>Auto Indent</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="ui_autoIndentEnabledCheckBox">
            <property name="text">
             <string>Enabled</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>83</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Style:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QRadioButton" name="ui_autoIndentIntelligentStyleRadioButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Intelligent</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="ui_autoIndentMaintainStyleRadioButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Maintain</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Indent Width:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="ui_autoIndentWidthSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="ui_miscGroup">
       <property name="title">
        <string>Misc</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="ui_callTipsCheckBox">
          <property name="text">
           <string>Call Tips</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ui_braceMatchingCheckBox">
          <property name="text">
           <string>Brace Matching</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ui_marginLineNumbersCheckBox">
          <property name="text">
           <string>Margin Line Numbers</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ui_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ui_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditorSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditorSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
